dnf/cli/main.py

```diff
@@ -123,13 +123,27 @@ def cli_run(cli, base):
             ret = resolving(cli, base)
         except dnf.exceptions.DepsolveError as e:
             ex_Error(e)
+            msg = ""
             if not cli.demands.allow_erasing and base._goal.problem_conflicts(available=True):
-                msg = _("(try to add '%s' to command line to replace conflicting "
-                        "packages") % "--allowerasing"
-                if cli.base.conf.strict:
-                    msg += _(" or '%s' to skip uninstallable packages)") % "--skip-broken"
+                msg += _("(try to add '%s' to command line to replace conflicting "
+                         "packages") % "--allowerasing"
+            if cli.base.conf.strict:
+                if not msg:
+                    msg += "(try to add "
```

Comment:
Sorry for noticing this late. This will not work for two reasons:

1. ```_()``` is missing (should be ```_("(try to add ")```).
2. This code is splitting and constructing sentence which sometimes may work but generally is considered a bad i18n habit. The possible branches are able to construct only 3 sentences:
   * ```"(try to add '%s' to replace conflicting...)"```
   * ```"(try to add '%s' to skip uninstallable...)"```
   * ```"(try to add '%s' to replace conflicting... or '%s' to skip uninstallable...)"```
What about writing all these sentences complete? Long but definitely unambiguous and translators will have no doubts about how to translate them. Also I bet that the number of _messages_ to translate will not be much larger or will even be smaller.

The same comment for the piece of code below, I am not repeating it.